lib/asciidoctor/js/postscript.rb

```diff
@@ -1,4 +1,3 @@
-require 'asciidoctor/converter'
```

Comment:
This line is only needed with Asciidoctor 1.5.5.
https://travis-ci.org/asciidoctor/asciidoctor.js/jobs/220190386

```
/home/travis/build/asciidoctor/asciidoctor.js/node_modules/opal-runtime/src/opal.js:4903
      throw exception;
      ^
Converter: uninitialized constant Asciidoctor::Converter
```